supervisor/homeassistant/api.py

```diff
@@ -17,6 +18,8 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+GET_CORE_STATE_MIN_VERSION: AwesomeVersion = AwesomeVersion("2023.8.0dev0")
```

Comment:
```suggestion
GET_CORE_STATE_MIN_VERSION: AwesomeVersion = AwesomeVersion("2023.8.0.dev20230720")
```
This needs to be the version that added it (or `2023.9.0dev0`), as `2023.8.0.dev20230718` is considered newer than `2023.8.0dev0` but does not contain this endpoint.

Reply:
Nice, that will do 👍